app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -284,6 +246,41 @@ private fun ColumnScope.EmailAddressesTextFields(selectedTransferType: () -> Tra
     }
 }
 
+@Composable
+private fun AdvancedOptions(
+    selectedTransferType: () -> TransferAdvancedSettingType?,
```

Comment:
Shouldn't this be called `selectedSetting` ?

Something about setting, and not about transfer.

Reply:
But it's about the transfer, not the settings.
Setting make it looks like it belongs to the application settings that can be find in the setting screen